### ISSUE TITLE
Warning if Chat Message Count Exceeds Safe Threshold

### ISSUE THREAD

**Issue by MrVauxs** [Dorako/pf2e-dorako-ui#154]
You can use the following bit of code to warn the user if the amount of chat messages in their game is above 100 and they need to refresh instead of waiting.

```js
if (game.messages.size > 100) {
    return ui.notifications.warn(game.i18n.localize("pf2e-dorako-ui.settings.warning"))
}
```

I am not making a PR due to the fact that every setting's `onChange` is a copy of the exact same function, which could be referred to with an actual function rather than copy pasted arrow function.

**Comment by Dorako** [comment]
I had an actual function, but something in the submenus is causing all onChanges to fire whenever anything changes, and I initially suspected it might be due to it being the same function or something, so I just made them entirely anonymous functions that happen to do similar things.
Regardless, a warning before the game locks up is probably an improvement.

**Comment by Dorako** [comment]
I implemented this in the new settings in 1.12.0
